edx/edx-enterprise
# Revise Data Sharing Consent page UI

@douglashall @brittneyexline FYI. 

**Description:** 
With the change to the main (track selection) Landing page, we will need to match the DSC experience.

Acceptance Criteria

1. Add Enterprise message to DSC page
2. Static text
3. Enterprise logo parameter
4. Enterprise name parameter
5. Site name parameter

**JIRA:** [ENT-352](https://openedx.atlassian.net/browse/ENT-352)

**Dependencies:** N/A

**Merge deadline:** 2-June-2017

**Testing instructions:**

1. visit https://ec-352.sandbox.edx.org/login
2. Click on `testshib_1` button
3. login with testshib credentials e.g. myself/myself
4. You can see the data sharing consent page with new design.

**Current UI Screen-shot**: 

<img width="1278" alt="screen shot 2017-05-25 at 6 12 51 pm" src="https://cloud.githubusercontent.com/assets/7334669/26451516/d017e6a0-4175-11e7-9c4f-3efdbe30b2a9.png">




**Merge checklist:**

- [x] Change log record added

**Post merge:**



#### NOTE: I will build a sandbox for UI testing.
